pkgs/development/python-modules/pytest-timeout/default.nix

```diff
@@ -15,10 +15,9 @@ buildPythonPackage rec {
     sha256 = "1cczcjhw4xx5sjkhxlhc5c1bkr7x6fcyx12wrnvwfckshdvblc2a";
   };
 
-  patches = fetchpatch {
-    url = "https://bitbucket.org/pytest-dev/pytest-timeout/commits/36998c891573d8ec1db1acd4f9438cb3cf2aee2e/raw";
-    sha256 = "05zc2w7mjgv8rm8i1cbxp7k09vlscmay5iy78jlzgjqkrx3wkf46";
-  };
+  patches = [
+    ./36998c891573d8ec1db1acd4f9438cb3cf2aee2e.patch
+  ];
 
   checkInputs = [ pytest pexpect ];
```

Comment:
Please use pytestCheckHook.

Reply:
Let's not change anything except for the patch URL so this doesn't cause any rebuilds.